### ISSUE TITLE
Dockerize application

### ISSUE THREAD

**Issue by karate** [anastasop/ostap#3]
I managed to install the application using docker, mostly to avoid installing stockfish locally.

Would you be interested in a PR?

**Comment by anastasop** [comment]
Keep it for now. My concern is that users of such applications like chess engines and are "happy" to have engines installed locally. If however this causes a burden in your workflow, we can merge the Dockerfile to speed things up.

**Comment by karate** [comment]
Thanks for the quick reply.
No need to merge it if you don't find it necessary. I can continue using it locally for development.

**Comment by anastasop** [comment]
Eventually we will need it. We need to find a better way to containerize engines than `Dockefile-stockfish` `Dockerfile-leila` etc If you some ideas on these pls submit the PR

**Comment by karate** [comment]
Not really. All I did is create a single alpine-based container, with python and stockfish, and make sure it can run `ostap`.

What's the problem with solutions like https://github.com/tchorwat/stockfish ?

**Comment by anastasop** [comment]
Gave it some thought, looks like engine integration is not straightforward. Please open your PR for stockfish.
